pandas-stubs/core/series.pyi

```diff
@@ -217,13 +217,43 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     @overload
     def __new__(
         cls,
-        data: IntervalIndex[IntervalT],
+        data: IntervalIndex[Interval[int]],
```

Comment:
I assume these changes are from #492

Reply:
Ugh. Same mistake again.  My local copy of `main` has the `Interval` change for some reason, and I keep branching off of that and making the same mistake.

This is now fixed for this PR.